### PR TITLE
[added] DropdownButton now supports block attribute

### PR DESCRIPTION
**Pull request by jguffey**
Hi there, Issue #1680 requested block property on Dropdown buttons, this PR attempts to incorporate that feature.

I've updated the documentation, and wrote a minimal test.

### CODE REVIEW

File: test/DropdownButtonSpec.js [hunk 1]
```diff
@@ -63,6 +63,21 @@ describe('DropdownButton', () => {
...
+      <DropdownButton title="blah" id="test-id" block vertical>
+        <MenuItem eventKey="1">Action</MenuItem>
+      </DropdownButton>
+    );
+    const menu = ReactTestUtils.findRenderedComponentWithType(instance, DropdownButton);
+    const node = ReactDOM.findDOMNode(instance);
+
+    menu.props.vertical.should.be.true;
+    menu.props.block.should.be.true;
+    console.log(node.className);
```
Comment (taion) [inline test/DropdownButtonSpec.js]: delete

**Comment by jguffey** [comment]
How's this look now?

**Comment by jquense** [comment]
`vertical` is only required when the dropdown `componentClass` is a `ButtonGroup`, which is the default case but not the only one. To most ppl its not going to make sense _why_ vertical is required since the rendering of a ButtonGroup is opaque to a user of DropdownButton, and just semantically its a button group of one. The other problem is that this won't work with SplitButton since there are two buttons in that component. I have no idea what the right behavior of NavDropdown would be either.

Which is all to say, I think this prop belongs in the DropdownButton component instead, be only the `block` prop and add `vertical` internally.

**Comment by jquense** [comment]
Gonna close for now, If you want to take a stab at moving this to DropdownButton @jguffey I'd be happy to review and merge

**Pull request closed without merging**